### ISSUE TITLE
No requests are captured by the HUNT scanner

### ISSUE THREAD

**Issue by juniorhero** [bugcrowd/HUNT#60]
I have observed that no requests are captured by hunt scanner
I have performed passive scan through Target tab in burp v2020.4

**Comment by Jineeshak** [comment]
same here, even with the remix version

**Comment by SimoneBovi** [comment]
Hi,
did you have specific targets in scope set within Burp?

Just targets in scope go through HUNT.

Cheers,
Simone

**Comment by bl4ckr0se-wq** [comment]
hey,
i have a (.*) regex in include scope and exclude scope contains some url.
is that a problem?

regards,
Trilok

**Comment by bl4ckr0se-wq** [comment]
hey,
i have a (.*) regex in include scope and exclude scope contains some url.
is that a problem?

regards,
Trilok

**Comment by cak** [comment]
@bl4ckr0se-wq  No, it should not be a problem. HUNT Scanner uses Burp's `callbacks.isInScope()` API, if the scope is set correctly HUNT will process those requests.
